stablehlo/transforms/Passes.td

```diff
@@ -36,7 +36,8 @@ def StablehloRefineShapesPass : Pass<"stablehlo-refine-shapes", "ModuleOp"> {
 
 def VhloLegalizeToStablehloPass : Pass<"vhlo-legalize-to-stablehlo", "ModuleOp"> {
   let summary = "Legalize VHLO to StableHLO.";
-  let dependentDialects = ["mlir::stablehlo::StablehloDialect", "mlir::shape::ShapeDialect", "mlir::quant::QuantizationDialect"];
+  let dependentDialects = ["mlir::func::FuncDialect", "mlir::stablehlo::StablehloDialect",
+                           "mlir::shape::ShapeDialect", "mlir::quant::QuantizationDialect"];
```

Comment:
By the way, this reminds me that we [added a bunch of dialects](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/Register.cpp) to `registerAllDialects` and then discovered that `dependentDialects` achieve the same effect. Can we revert `registerAllDialects` given that?

Reply:
Reverted to just `registry.insert<mlir::sparse_tensor::SparseTensorDialect>();`